kubectl-intermediate/deploy.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
-# Change directory to the home user
-
-cd ~/
-
-kubectl create -f  label-deploy.yaml
-
-
-
-
+set +v # suppress echoing the echo command being invoked
+clear
+echo "Please wait a moment while we apply some deployments to the frontend namespace.."
+kubectl create -f ~/label-deploy.yaml
```

Comment:
Since the message it put in about waiting, how about adding a [`kubectl wait`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait) command after so that way folks know when it's actually completed?

Reply:
i considered using kubectl wait, but the "--help" said the command was experimental so i decided against it.
i think what i have here is better than what was there before, where sometimes invoking some of the lab instructions would happen too early, before the deployment was created.